cfme/tests/infrastructure/test_providers.py

```diff
@@ -492,3 +495,56 @@ def test_infra_discovery_screen(appliance):
                    "to_ip4": ips['to']})
         view.start.click()
         view.flash.assert_message(ips['msg'])
+
+
+@pytest.fixture
+def setup_provider_min_templates(request, appliance, provider, min_templates):
+    templates_yaml = len(provider.data.get('templates', {}))
+    if templates_yaml < min_templates:
```

Comment:
I think you'll want to skip this check for this test context, as the number of templates in the yaml is almost always going to be lower than the actual number of templates available. We only define in the yaml the templates that we use for provisioning tests, and I think this ends up too restrictive - currently skipping for 4 templates against both versions of rhv and scvmm.

Reply:
Updated as requested.